### DISCUSSION TITLE
ARMv8-A support

### DISCUSSION THREAD

**Discussion by rpmsousa** [ARM-software/CMSIS_5#1311] [Q&A]
Hi,

I would like to know if there already any plans to add ARMv8-A support in CMSIS project? And as a follow up question, would patches be accepted to add (basic) support to existing CMSIS/Cortex_A/*.[ch] files?

Thanks,
Rui Sousa

**Comment by JonatanAntoni** [comment]
Hi @rpmsousa,

currently we do not have plans to add Armv8-A support to CMSIS-Core(A).
But contributions are always welcome.

Thanks,
Jonatan
